include/odp/api/spec/traffic_mngr.h

```diff
@@ -247,6 +247,75 @@ typedef struct odp_tm_queue_stats_capability_t {
 	};
 } odp_tm_queue_stats_capability_t;
 
+/** TM node specific statistics counters */
```

Comment:
I'd like to avoid having to repeat `sent through this tm node` for each struct member. If required, an additional explainer sentence could be added here. See comments below.